NEW_PROJECT_APPLICATION.md

```diff
@@ -13,12 +13,19 @@ A rough description of the project in less than 100 words.
 Please refer to [the Cross Project Council's Charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md).
 Please keep your answer to less than 250 words.
 
+## Impact and users of the project
+- What is the ecosystem impact of the project? 
+- Who are the biggest users of the project? (_if they are not JavaScript space, then will it ever expand to JavaScript space_)
+- What is the potential long term impact and adoption? 
+- How does it differ from other projects?
+
 ## List of all repos that are part of the project
 
 For each repository, please provide:
 
 - link to repo,
 - license information,
+- who owns the IP
```

Comment:
Isn't this part of the license?

Reply:
I don't think so. But certain project's IP might be tied to their employer and it will be good to know this at the initial stage itself.

Reply:
I agree that asking this question at this point is important. I would suggest saying `Intellectual Property` as opposed to just `IP` to be clear.